package.json

```diff
@@ -51,6 +52,6 @@
     "typescript": "^4.5.4"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=14.0.0"
```

Comment:
What minimum version do the other `aio libs` support?

Reply:
aio app builder also expects min node version as `14.0`, so this should support other `aio libs`.

Reply:
Shouldn't we change this to >=12.0.0  based on the status checks we have?